### PR TITLE
Update WPF getting started for .NET 6.0 VS2022

### PR DESCRIPTION
**Pull request by adegeo**
- Moved images/code to moniker folders
- Added VS 2022 screenshots

### CODE REVIEW

**Comment by adegeo** [comment]
@tdykstra ready for review

File: dotnet-desktop-guide/net/wpf/get-started/create-app-visual-studio.md [hunk 1]
```diff
@@ -58,19 +74,64 @@ The first step to creating a new app is opening Visual Studio and generating the
     01. Optionally, choose a different **Location** to save your code.
     01. Select the **Next** button.
 
-    :::image type="content" source="media/create-app-visual-studio/vs-config-new-project.png" alt-text="Configure new WPF project in Visual Studio 2019 for .NET":::
+    :::image type="content" source="media/create-app-visual-studio/netdesktop-5.0/vs-config-new-project.png" alt-text="Configure new WPF project in Visual Studio 2019 for .NET":::
```
Comment (tdykstra) [inline dotnet-desktop-guide/net/wpf/get-started/create-app-visual-studio.md:77]: Period for alt text here and in other places.
```suggestion
    :::image type="content" source="media/create-app-visual-studio/netdesktop-5.0/vs-config-new-project.png" alt-text="Configure new WPF project in Visual Studio 2019 for .NET.":::
```

File: dotnet-desktop-guide/net/wpf/get-started/create-app-visual-studio.md [hunk 2]
```diff
@@ -121,7 +182,7 @@ The document root `<Window>` represents the type of object being described by th
 
 - Namespaces
 
-  An XML namespace provides structure to the XML, determining what is or isn't allowed to be declared in the file.
+  An XML namespace provides structure to the XML, determining XML content can be declared in the file.
```
Comment (tdykstra) [inline dotnet-desktop-guide/net/wpf/get-started/create-app-visual-studio.md:185]:
```suggestion
  An XML namespace provides structure to the XML, determining what XML content can be declared in the file.
```

File: dotnet-desktop-guide/net/wpf/get-started/create-app-visual-studio.md [hunk 3]
```diff
@@ -131,17 +192,17 @@ The document root `<Window>` represents the type of object being described by th
 
 - `Title` attribute
 
-  Any normal attribute declared on the XAML object sets a property of that object. In this case the `Title` attribute sets the `Window.Title` property.
+  Any normal attribute declared on the XAML object sets a property of that object. In this case, the `Title` attribute sets the `Window.Title` property.
 
 ## Change the window
 
 First, let's run the project and see the default output. You'll see that a window that pops up, without any controls, and a title of **MainWindow**:
```
Comment (tdykstra) [inline dotnet-desktop-guide/net/wpf/get-started/create-app-visual-studio.md:199]:
```suggestion
First, run the project and see the default output. You'll see a window that pops up, without any controls, and a title of **MainWindow**:
```

**Review by tdykstra — APPROVED** [review]
LGTM

**Pull request merged**